src/core_modules/capture-core/components/WidgetTwoEventWorkspace/WidgetWrapper/WidgetWrapper.container.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+import React from 'react';
+import { colors, spacersNum, IconLink16 } from '@dhis2/ui';
+import i18n from '@dhis2/d2-i18n';
+import { withStyles } from '@material-ui/core/';
+import type { Props } from './WidgetWrapper.types';
+import { WidgetTwoEventWorkspaceWrapperTypes } from '../index';
+
+const styles = {
+    container: {
+        marginBottom: '16px',
+        margin: '16px',
+    },
+    header: {
+        display: 'flex',
+        alignItems: 'center',
+        paddingBottom: spacersNum.dp16,
+        fontWeight: 500,
+        fontSize: 16,
+        color: colors.grey800,
+    },
+    referalResponse: {
+        padding: spacersNum.dp16,
+        backgroundColor: colors.blue100,
+        borderRadius: '3px',
+    },
+    linkedEvent: {
+        color: colors.blue600,
+        verticalAlign: 'middle',
+        display: 'flex',
+        fontSize: '16px',
+        fontWeight: '500',
+    },
+    icon: {
+        marginRight: spacersNum.dp8,
+    },
+    decription: {
+        margin: `${spacersNum.dp8}px 0`,
+    },
+};
+
+const WidgetWrapperPlain = ({ widget, type, classes }: Props) => {
+    if (type === WidgetTwoEventWorkspaceWrapperTypes.EDIT_EVENT) {
```

Comment:
Are there any plans of having multiple TwoEventWorkspaces? Is this to support a standalone widget in the future?

Reply:
Yes the `type` logic is only to support a standalone widget